_posts/2022-04-15-flotta-and-raspberry-pi.markdown

```diff
@@ -18,6 +18,10 @@ sensors deployment, vehicle deployment or many other cases.
 
 Let's learn how Flotta can be used to manage it by following steps described in this article.
 
+## Prerequisite
+
+- Installed Flotta-Operator with a version higher than v0.1.0.
```

Comment:
[Line 72]( https://github.com/project-flotta/project-flotta.github.io/pull/81/files#diff-d95797b19acfff923b52589a76f257651779f277e462d6c5294ecce067409737R72) in this file refers to the same issue.
Could you remove that line to avoid duplication?

Reply:
Oh good point. The link seems to be broken too. 
`The 'project-flotta/flotta-operator' repository doesn't contain the 'docs/user-guide/running.md' path in 'main'.`